op-tests/test-blspy-g2.txt

```diff
@@ -0,0 +1,120 @@
+bls_g2_add 0x8c4441872d3b5ea5d69a82829d910263eba8e4ef5bb0e1cd475b6ebb0267b885c3ae73ec840fb5234d669aa8df5369a2018c55abc032796bc847fb806db92508872a5e3d225427d6bccfdac274297bbea554edd6c3ca13d758d9e79cf293d113 0x84250bba4dc02a41dfbb4913386abb67b2b03823a246aa8d7aab78e547469552746cda9e9da25fc867fe0109219b5b1618d5941e1298231220bf93267ebe984fc70d1969afdf0bdf61886c325d316b77dc4c3516fb02c7827db5f819218c6ab2 => 0x95857a3a5cc48db0557d5ae6465fe003116c685bab95c059ab0d91667b0442a3a451ffa10698e571988d1cffe692941015815e4b0f20bf1cbf3c1c298a0c24c2d521ff3892e1232bf237a5f30fcc4d05889c77de21f0ffb577d8afbfec38481e | 11135562
+bls_g2_add 0x95857a3a5cc48db0557d5ae6465fe003116c685bab95c059ab0d91667b0442a3a451ffa10698e571988d1cffe692941015815e4b0f20bf1cbf3c1c298a0c24c2d521ff3892e1232bf237a5f30fcc4d05889c77de21f0ffb577d8afbfec38481e 0xaab6a407228638c872bd60e4112a5b4ae1ee15ff5123ae6a762dc97cd2184dc0cf6aee27db0221454c662138c93d369e094cf1ecf824f2c1fa374dd302c33f62d4497cc6cf04c6fcf0f4da811f2f72dec379629a1b370eb746db0b9447ecf74b => 0x9661ca919fbbf0022cf493691a0fc5d14e3ce66c5ffbbd3e20b1d8f5f9e96984fb54777d8ef5f3199b8def4e58423e9c1933930cfacee37267b7c06e049eb8df55ad86ba03b9bdb6d7ff30e8259a65f6f318e47ef24578dd5d514c4554935cad | 11135562
+bls_g2_add 0x9661ca919fbbf0022cf493691a0fc5d14e3ce66c5ffbbd3e20b1d8f5f9e96984fb54777d8ef5f3199b8def4e58423e9c1933930cfacee37267b7c06e049eb8df55ad86ba03b9bdb6d7ff30e8259a65f6f318e47ef24578dd5d514c4554935cad 0x8e112b9b019c4c83475ad272f96496e85497a689b14bd8fd031a6fb2247217a501d14152d68a17bfc8860baec4cc126013641b9626dacd761aa54cf11c87542433f855a735ca790ae8f154db815b99b0c155065b5169729f522c0698c9f29cd0 => 0x85e3bdc96b539f17a952e185b0868bea9548c176e481e8a37dd33789af0c7c9599300c5fe991ca9f9e76f0c6f2c3367e17c0eb4d094c73d2488d9b15eb989c6eb31c8b94cd4334aa7dafcd968e4d0fbd6e80efd791c9a89e5815a40ae681852c | 11135562
+bls_g2_subtract 0x96bbcfbca241004ba03968b7433664c1b944245828af7a1ebda771495c90d4816508d65db40b601bce51a3689c251f731699411b56e2172dd82eb96f571a691ba2984f1e4e39a05646ceb9c82d1108e78fe3fdfd94cff4fb447efe498dcecc25 0xa35068bd876cd804a57e9695ad9585271b20f2a67c0f70a05ec47c597a6c3754083d1a974b6bf9a90c1e4e5208b73d7a120ce4190f328a3b90a15c2c7a7973a5894a8e16a63c36adb4e4995d165192d9b8c26fc15a84655c90a92b846f7952fa => 0xb681d21f5b5f31c5dc5a7983992c3be169cbd684a84f4d1ffe0c0c21923f7a023bc57ed4d3c445a3c9709203104e16b5091abec762e3ebc5edc5e7f170a2be4f71e868dc25b90b4c9dd9952edaaff163a177acf977f36e5d66f485fd94962428 | 11137794
```

Comment:
0 & 1 parameter

Reply:
And more than 2 parameters.

Reply:
the no argument case is covered here: https://github.com/Chia-Network/clvm_rs/pull/274/commits/83ff075a5957ebb89fa5f64ff9fe9aaf208b3aa3#diff-8e926fa1adc34a468edbbfcb76a3996682c288dc76919bad5a6876cbe68fde84R75
I'll add the single argument case next to it